src/github.com/getlantern/flashlight/client/balancer.go

```diff
@@ -55,26 +44,15 @@ func (client *Client) initBalancer(cfg *ClientConfig) {
 	}
 
 	bal := balancer.New(dialers...)
-
-	if client.balInitialized {
-		log.Trace("Draining balancer channel")
-		old := <-client.balCh
+	oldBal, ok := client.bal.Get(25 * time.Millisecond)
```

Comment:
Oh, I see what you mean though.  In this particular case, if we don't get anything immediately, there's nothing to get.  Yes, I'll change it :)